Specs/createGeometryUpdaterGroundGeometrySpecs.js

```diff
@@ -154,6 +156,24 @@ define([
             dynamicUpdater.destroy();
             updater.destroy();
         });
+
+        it('dynamic updater on terrain propagates classification type', function() {
+            var entity = createDynamicEntity();
```

Comment:
Instead of setting `classificationType` in each of the `XXXGeometryUpdaterSpec` files, do `entity[geometryPropertyName].classificationType = ClassificationType.BOTH` in this file